internal/platform/writer/printers.go

```diff
@@ -101,6 +101,13 @@ func requestPrinter() pterm.PrefixPrinter {
 	}
 }
 
+func normalPrinter() pterm.PrefixPrinter {
```

Comment:
Nit; I'm not sure whether we really need this function or not, cause it has no prefix, I think you can just initialize it where used.

Reply:
removed it